src/lib/isBefore.js

```diff
@@ -1,9 +1,12 @@
 import assertString from './util/assertString';
 import toDate from './toDate';
 
-export default function isBefore(str, date = String(new Date())) {
-  assertString(str);
-  const comparison = toDate(date);
-  const original = toDate(str);
-  return !!(original && comparison && original < comparison);
+export default function isBefore(date, options = {}) {
+  assertString(date);
+
```

Comment:
```suggestion
```

The string already gets asserted in the `toDate` function

Reply:
Removed :).